skrub/_reporting/_data/templates/_variables.css

```diff
@@ -80,11 +82,11 @@
     /* borders */
     --color-border-primary: #aaa;
     --color-border-secondary: #bfbfbf;
-    --color-border-highlight: #9fce9f;
+    --color-border-card: #9fce9f;
 
     /* shadows */
     --color-shadow: hsl(0, 0%, 63%);
-    --color-shadow-keyboard: rgb(255, 255, 255, 0.5);
+    --color-shadow-keyboard: var(--color-border-secondary);
```

Comment:
that one is not related to this pr but I noticed it and figured let's fix it here and not do another pr for 2 lines of diff